lib/index.js

```diff
@@ -25,12 +25,17 @@ exports.register = (server, options, next) => {
     options.redisClient.multi();
     options.redisClient.set(key, 0, 'EX', rate.window, 'NX'); // if key not found, insert key:0 with window seconds expiry
     options.redisClient.incr(key);
+    options.redisClient.ttl(key);
+
     return options.redisClient.exec()
     .then((results) => {
-      rate.current = results[1]; // result of 2nd redis instruction
+      const remaining = rate.limit - results[1]; // result of 2nd redis instruction
```

Comment:
since you dont comment the use of `results[2]` down below, i dont think you need this comment